docs/source/tutorial/making_predictions.rst

```diff
@@ -29,19 +30,24 @@ which will already be in memory. Each of the high-level interfaces are exposed t
 model:
 
 >>> from pykeen.pipeline import pipeline
+>>> from pykeen.models import predict
 >>> # Run the pipeline
 >>> result = pipeline(dataset='Nations', model='RotatE')
 >>> model = result.model
 >>> # Predict tails
->>> predicted_tails_df = model.get_tail_prediction_df('brazil', 'intergovorgs', triples_factory=result.training)
+>>> predicted_tails_df = predict.get_tail_prediction_df(
+...     model, 'brazil', 'intergovorgs', triples_factory=result.training)
 >>> # Predict relations
->>> predicted_relations_df = model.get_relation_prediction_df('brazil', 'uk', triples_factory=result.training)
+>>> predicted_relations_df = predict.get_relation_prediction_df(
+...     model, 'brazil', 'uk', triples_factory=result.training)
 >>> # Predict heads
->>> predicted_heads_df = model.get_head_prediction_df('conferences', 'brazil', triples_factory=result.training)
+>>> predicted_heads_df = predict.get_head_prediction_df(model, 'conferences', 'brazil', triples_factory=result.training)
 >>> # Score all triples (memory intensive)
->>> predictions_df = model.get_all_prediction_df(triples_factory=result.training)
+>>> predictions_df = predict.get_all_prediction_df(model, triples_factory=result.training)
 >>> # Score top K triples
->>> top_k_predictions_df = model.get_all_prediction_df(k=150, triples_factory=result.training)
+>>> top_k_predictions_df = predict.get_all_prediction_df(model, k=150, triples_factory=result.training)
+>>> # Score training triples
+>>> score_df = predict.predict_triples_df(model=model, triples=result.training.mapped_triples, triples_factory=result.training)
```

Comment:
here now could also use the simpler variant, where a (list of) string triple(s) is provided.

Reply:
that's a good idea - since this is the high level user facing function we should make it able to take a wide variety of things

Reply:
@mberr if you want to make more updates for this, great. Otherwise the PR is ready